CONTRIBUTING.md

```diff
@@ -0,0 +1,200 @@
+# Contribution Guide
+
+Contributions are highly welcomed and appreciated. Every little bit helps,
+so do not hesitate! You can make a high impact on `xskillscore` just by using it and
+reporting [issues](https://github.com/raybellwaves/xskillscore/issues).
+
+The following sections cover some general guidelines
+regarding development in `xskillscore` for maintainers and contributors.
+
+Nothing here is set in stone and can't be changed.
+Feel free to suggest improvements or changes in the workflow.
+
+* [Feature Requests and Feedback](#Feature-Requests-and-Feedback)
+* [Bug Reports](#Bug-Reports)
+* [Fix Bugs](#Fix-Bugs)
+* [Write Documentation](#Write-Documentation)
+* [Preparing Pull Requests](#Preparing-Pull-Requests)
+
+## Feature Requests and Feedback
+
+We are eager to hear about your requests for new features and any suggestions about the
+API, infrastructure, and so on. Feel free to submit these as
+[issues](https://github.com/raybellwaves/xskillscore/issues/new?assignees=&labels=feature+request&template=feature_request.md) with the label "feature request."
```

Comment:
Note we don't have these templates. This may have been a bug before. we just have https://github.com/raybellwaves/xskillscore/blob/master/.github/ISSUE_TEMPLATE/ISSUE.md